tests/meta/report/test_rnafusion_api.py

```diff
@@ -0,0 +1,60 @@
+"""Test module for the Rnafusion delivery report API."""
+from typing import Dict
+
+from cg.models.report.metadata import RnafusionSampleMetadataModel
+from cg.models.rnafusion.analysis import RnafusionAnalysis
+
+from cg.meta.report.rnafusion import RnafusionReportAPI
+
+from cg.models.cg_config import CGConfig
+
+from cg.meta.workflow.rnafusion import RnafusionAnalysisAPI
+
+from cg.store.models import Sample, Family
+
+from tests.store_helpers import StoreHelpers
+
+from tests.cli.workflow.rnafusion.conftest import (
```

Comment:
It would be great to place the fixtures in a conftest file accessible to the tests using them instead of using imports

Reply:
Yes I agree, I'll move it upstream and refactor the inconsistent fixture names. Would laso be good to add some test to the RnafusionAnalysisAPI as only the CLI is currently tested...